manager/controllers/packages.go

```diff
@@ -16,11 +16,12 @@ var PackagesOpts = ListOpts{
 	DefaultSort:    "name",
 }
 
+// nolint: lll
 type PackageItem struct {
 	Name             string `json:"name" query:"pn.name"`
 	SystemsInstalled int    `json:"systems_installed" query:"count(sp.id)"`
 	SystemsUpdatable int    `json:"systems_updatable" query:"count(sp.id) filter (where spkg.update_data is not null)"`
-	// Description      string `json:"description"`
+	Summary          string `json:"summary" query:"(select s.value from package p inner join strings s on p.summary_hash = s.id inner join advisory_metadata am on p.advisory_id = am.id where p.name_id = pn.id order by am.public_date limit 1)"`
```

Comment:
Funny solution, ok it's working. Is it at least possible to split the query text into multiple lines to be more clear?

Reply:
Tried, however it wasn't accepted by the parser for `query` attribute